### PR TITLE
Major refinements in the intro

### PR DESCRIPTION
**Pull request by ypriverol**
This PR addressed major refinements and comments in the intro from @bgruening @danielskatz @bittremieux. Please take a look and let me know what you think. You can add your comments inline.

- [ ] The first paragraph was refined to remove numbers of %, but the original studies claiming the need for research software were added. 
- [ ] Refine the challenges by removing the one with more comments (@danielskatz @bittremieux )and adding a new one about reproducibility.   
- [ ] References were added to all major paragraphs. 

Thanks to all who have contributed to the intro.

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -2,31 +2,39 @@
...
+- Ensuring reproducibility is a challenge due to the limited access to detailed algorithmic information, which hinders validation and extension of methods.
 
 - Although most software tools are described in publications, the absence of open-source code, comprehensive documentation, and version control often impedes reproducibility and reuse. 
-This lack of accessibility also limits opportunities for computer scientists [maybe this should be researchers?] and developers to contribute effectively to advancing proteomics software.
+This lack of accessibility also limits opportunities for researchers and developers to contribute effectively to advancing proteomics software. 
+- Custom licenses and restrictions on software distribution can further complicate the situation, making it difficult to share, modify, or redistribute software, and hindering the development of a collaborative and open-source ecosystem.
+Proteomics software is usually on distribution by licences and also platform specific, which makes it difficult to use in a diverse environment and services, putting a restriction to the field and the integration of proteomics with other omics fields.
```
Comment (timosachsenberg) [inline manuscript.md]: Not sure how to understand "platform specific" in this context.
OS? or more like a software that integrates other tools (e.g., FragPipe)

Comment (ypriverol) [reply manuscript.md]: @timosachsenberg can you read it now? Does it read better?

File: manuscript.md [hunk 2]
```diff
@@ -2,31 +2,38 @@
...
+The community has tackled this challenge with CI/CD pipelines, as for example, described in 10.1101/gr.276963.122, but this relies on OSS and the permission to freely redistribute software along the entire dependency chain. 
+If one piece in this supply chain is not OS, exceptions need to be handled, automatization is harder or impossible. 
+In sum, it creates a huge additional burden for the entire community downstream of the non-OS software package, which on its own is a hidden cost.
 
-OSS has demonstrated clear benefits in increasing the accessibility, reusability, and visibility of scientific software [REF?]. 
+Open-source software (OSS) and making the FAIR principles (Findable, Accessible, Interoperable, Reusable) offers a solution to these challenges, empowering proteomics with tools that are not only accessible but also foster community-driven development, rigorous validation, and transparent sharing of methodologies [].
```
Comment (wfondrie) [inline manuscript.md]: Rephrase to: 
> Open-source software (OSS) following FAIR principles ...

**Review by wfondrie — APPROVED** [review]
LGTM! I left one minor suggestion.

File: manuscript.md (hunk 2 above)
Comment (ypriverol) [reply manuscript.md]: Changed.

**Pull request merged**